### DISCUSSION TITLE
Guys, Can someone advise me what to do with quality of drawble resources?

### DISCUSSION THREAD

**Discussion by abrabr21** [microsoft/WSA#89] [General]
I have a problem with quality of drawbles resources in my project.
WSA takes two types of resources : hdpi (sw590, sw700) - for window mode, xhdpi (sw780) - for full screen.
Is there any opportunity may be to add special drawble folder for WSA like xxxhdpi or smth like that.
Quality of drawbles in sw780 for full screen is not enough.

**Comment by nieubank** [comment]
This is great idea, could you file a feature request in Issues?

**Reply by abrabr21** [reply]
Already done it.
